api/v1alpha1/types_etcd.go

```diff
@@ -381,16 +383,23 @@ type EtcdStatus struct {
 	// +optional
 	ServiceName *string `json:"serviceName,omitempty"`
 	// LastError represents the last occurred error.
+	// Deprecated: Use LastErrors instead.
 	// +optional
 	LastError *string `json:"lastError,omitempty"`
+	// LastErrors captures errors that occurred during the last operation.
+	// +optional
+	LastErrors []LastError `json:"lastErrors,omitempty"`
+	// LastOperation indicates the last operation performed on this resource.
+	// +optional
+	LastOperation *LastOperation `json:"lastOperation,omitempty"`
```

Comment:
`LastOperation` talks about the last operation performed. It's not really a "condition" on the etcd cluster, since it doesn't indicate a state or partial state of the cluster (like readiness of backup readiness), but rather indicates an operation that was performed in the recent past. Hence, it requires its own API field under `Status`.